plugins/grype/surfactantplugin_grype.py

```diff
@@ -69,24 +90,51 @@ def run_grype(filename: str) -> object:
 def extract_file_info(
     sbom: SBOM, software: Software, filename: str, filetype: str, children: list
 ) -> Optional[List[Software]]:
-    if disable_plugin or filetype not in ("DOCKER_TAR", "DOCKER_GZIP"):
+    logger.info(f"Extracting file info for {filename} of type {filetype}")
+
+    if disable_plugin:
+        logger.warning("Grype plugin is disabled. Skipping file analysis.")
```

Comment:
Same with this message -- I'd expect it to get printed for every single file when grype is disabled.